src/main/java/tc/oc/pgm/rotation/RotationManager.java

```diff
@@ -0,0 +1,220 @@
+package tc.oc.pgm.rotation;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.apache.commons.io.FileUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.configuration.file.FileConfiguration;
+import org.bukkit.configuration.file.YamlConfiguration;
+import org.bukkit.entity.Player;
+import tc.oc.pgm.AllTranslations;
+import tc.oc.pgm.api.PGM;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.api.match.MatchManager;
+import tc.oc.pgm.map.PGMMap;
+
+public class RotationManager implements PGMMapOrderProvider {
+
+  private MatchManager matchManager;
+
+  private File rotationsFile;
+  private List<Rotation> rotations = new ArrayList<>();
+  private Rotation activeRotation;
+  private boolean isEvaluatingPlayerCount = true;
```

Comment:
What does this mean? Name is not clear, is this dynamic rotations?

Reply:
Should be clearer now, with the classes being refactored. https://github.com/Electroid/PGM/pull/114/commits/fe83e3a18ec9419e8184475a0ccd8c4fb5a5e93a